WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/logic/interval/intervaltypes/IntervalBase.java

```diff
@@ -138,4 +153,38 @@ public Object clone() throws CloneNotSupportedException {
 		return super.clone();
 	}
 
+	/**
+	 * Checks whether the parameter is an IntervalBase and is equal to this by
+	 * comparing the start dates, end dates, types and session seeds of the two
+	 * intervals.
+	 */
+	@Override
+	public boolean equals(Object obj) {
+		if (this == obj)
+			return true;
```

Comment:
No worries, they are not there because this method was generated automatically, but I'll fix it right away